modules/statistics/statistics.js

```diff
@@ -104,23 +106,32 @@ function Statistics(xmpp, options) {
             && (Statistics.disableThirdPartyRequests !== true);
     if (this.callStatsIntegrationEnabled) {
         loadCallStatsAPI(this.options.callStatsCustomScriptUrl);
+
+        if (!this.options.callStatsConfIDNamespace) {
```

Comment:
yes, but are you ok with logging this or should this go away ? I think that fixing the jitsi-meet part is another PR